pandas/core/ops/docstrings.py

```diff
@@ -233,6 +233,60 @@ def _make_flex_doc(op_name, typ):
 dtype: float64
 """
 
+_eq_example_SERIES = """
+Examples
+--------
+s1 = pd.Series([1, 2, 3, 4, 5])
+s2 = pd.Series([1, 2, 5, 9, 2])
```

Comment:
Besides the missing `>>>` (see other docstrings for reference), do you mind making the examples more concise please? I think two `Series` with values `[1, 2, 3]` and `[1, 9]` should be enough to show all the cases (without `fill_value` and with `fill_value=3`).